lib/bundler/plugin/index.rb

```diff
@@ -29,7 +29,12 @@ def initialize
         @hooks = {}
         @load_paths = {}
 
-        load_index(global_index_file, true)
+        begin
+          load_index(global_index_file, true)
+        rescue GenericSystemCallError
```

Comment:
We probably just want to catch `Errno::EROFS` instead of `GenericSystemCallError` because there are errors we do want to catch, such as when the user not have the correct permissions, disk if full etc.

Reply:
the error being surfaced is Bundler::GenericSystemCallError, rescuing anything else won't work ?

Reply:
Oh, i got confused between `GenericSystemCallError` and `SystemCallError`, sorry.